kubernetes/overlays/authorization-updater/dev-refactor/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  KAFKA_CLIENT_ID: "dev-pagopa-interop-authorization-updater"
+  KAFKA_GROUP_ID: "authorization-updater"
+  KAFKA_BROKERS: {{KAFKA_BROKERS}}
+  KAFKA_TOPICS: "event-store.catalog.events,event-store.agreement.events"
+  AUTH_MNGT_SERVICE_URL: "interop-be-authorization-management.dev-refactor.svc.cluster.local"
```

Comment:
urls could go under a new common configmap to have all of them just in one place?

Reply:
Good catch, we already have [this configmap](https://github.com/pagopa/pdnd-interop-platform-deployment/blob/1.0.x/kubernetes/commons/be-refactor/dev-refactor/configmap.yaml) for dev-refactor.
Do you think it's okay to use that one or do we split it into more configmaps?

Reply:
in my mind it seems cleaner to have one configmap per context, instead of one big configmap with everything.
For example one for the read model, one for the event store, one for the services urls, etc.
But let me know what you think

Reply:
Makes sense. Since the change will impact multiple overlays, I will do it in another PR. I'll merge this one for the moment. Thank you!